docs/core_docs/docs/modules/data_connection/retrievers/custom.mdx

```diff
@@ -0,0 +1,73 @@
+---
+hide_table_of_contents: true
+sidebar_position: 0
+---
+
+# Custom retrievers
```

Comment:
Might be worth linking the general interface or retriever if a page exists (query in, list of documents out)